src/main/java/ChatRequest.java

```diff
@@ -0,0 +1,27 @@
+import lombok.*;
+
+import java.util.List;
+
+    /**
+     * @author madhankumar
```

Comment:
Author tag is not recommended as it can easily get outdated and should be avoided for maintenance reasons. (Severity: low)